src/codegen/templates/_task_modules/triggering/start_trigger.py.mako

```diff
@@ -3,7 +3,7 @@
     from codegen.utilities.function_helpers import get_functions, get_enums_used as functions_enums
     from codegen.utilities.attribute_helpers import get_attributes,  get_enums_used as attribute_enums
     from codegen.utilities.helpers import get_enums_to_import
-    attributes = get_attributes(data, "StartTrigger")
+    attributes = get_attributes(data, "StartTrigger", "TimeStartTrigger")
```

Comment:
You shouldn't have to do this here. Scrapigen should translate TimeStartTrigger to StartTrigger like it does for DigitalEdgeStartTrigger, AnalogWindowStartTrigger, etc.

Reply:
I have updated scrapigen to correctly generate StartTrigger instead of TimeStartTrigger